appcenter-analytics/package.json

```diff
@@ -36,9 +36,20 @@
     "eslint-config-airbnb": "^15.1.0",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-react": "^7.2.1"
+    "eslint-plugin-react": "^7.2.1",
+    "jest": "^23.6.0",
+    "react": "^16.6.3",
```

Comment:
This will update react native for the application because of the SDK? This will have side effects on applications using older versions of React Native, and React Native introduces breaking changes.

Reply:
It is a dev dependency, it won't have any effect for users of the sdk

Reply:
Like @bmourat said, dev dependency shouldn't be a problem. In case sdk users don't want to conflict RN version, they can use `npm install --production` to skip installing dev dependencies.

Reply:
Most people probably don't know about `--production`, please test on an app that uses old react native version like TestApp34 and see what happens without using that flag.